examples/fluids/qfunctions/turb_stats_types.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#ifndef turb_stats_types_h
+#define turb_stats_types_h
+
+#include "./newtonian_types.h"
+
+enum {
```

Comment:
Naming it will improve compiler diagnostics.
```suggestion
enum TurbComponent {
```